e2e/test/Helpers/digitaltwins/ThermostatTwin.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Probably should be under a folder called `models`, not helpers.

Reply:
sure!